### PR TITLE
Remove Type Body Fingerprints Flags

### PR DESCRIPTION
**Pull request by CodaFi**
Remove the staging flags for type body fingerprints.

### CODE REVIEW

**Comment by CodaFi** [comment]
@swift-ci test

**Comment by CodaFi** [comment]
@swift-ci test source compatibility

**Comment by CodaFi** [comment]
@swift-ci test

**Comment by CodaFi** [comment]
@swift-ci test

**Comment by slavapestov** [comment]
Excellent! Can you file a radar for the code completion thing?

File: lib/Parse/ParseDecl.cpp [hunk 1]
```diff
@@ -4587,7 +4592,7 @@ Parser::parseDeclList(SourceLoc LBLoc, SourceLoc &RBLoc, Diag<> ErrorDiag,
   if (RBLoc.isInvalid())
     hadError = true;
 
-  if (!Context.LangOpts.EnableTypeFingerprints)
+  if (L->isCodeCompletion())
```
Comment (davidungar) [inline lib/Parse/ParseDecl.cpp:4595]: This looks suspicious, because the predicate on line 4559 is different. Should this one be the same?

**Review by davidungar — COMMENTED** [review]
Aside from my "this looks suspicious" comment, it looks fine!

**Comment by CodaFi** [comment]
@swift-ci test Linux

File: lib/Parse/ParseDecl.cpp (hunk 1 above)
Comment (slavapestov) [reply lib/Parse/ParseDecl.cpp:4595]: My guess is that previously code completion had EnableTypeFingerprints disabled, so this brings back the old behavior but only when performing code completion. I already suggested Robert file a radar to investigate this further in my review above.

**Comment by CodaFi** [comment]
rdar://69890297

**Comment by CodaFi** [comment]
👍

**Comment by CodaFi** [comment]
⛵

**Pull request merged**